gov/agendas/deployments.go

```diff
@@ -122,42 +124,37 @@ func agendasForVoteVersion(ver uint32, client DeploymentSource) (agendas []Agend
 		})
 	}
 
-	return
-}
-
-// IsAgendasAvailable checks for the availabily of agendas in the db by vote
-// version.
-func (db *AgendaDB) isAgendasAvailable(version uint32) bool {
-	agenda := make([]AgendaTagged, 0)
-	err := db.sdb.Find("VoteVersion", version, &agenda, storm.Limit(1))
-	if len(agenda) == 0 || err != nil {
-		return false
-	}
-
-	return true
+	return agendas, nil
 }
 
-// updatedb used when needed to keep the saved db up-to-date.
-func (db *AgendaDB) updatedb(voteVersion uint32, client DeploymentSource) int {
+// updatedb checks if vote versions available in chaincfg.ConsensusDeployment
+// are already updated in the agendas db, if not yet their data is updated.
+// dcrjson.GetVoteInfoResult and chaincfg.ConsensusDeployment hold almost similar
```

Comment:
`chaincfg.Params.Deployments` was a great find but It is missing some important data like votes `status` that is easily returned in `dcrjson.GetVoteInfoResult` by `GetVotesVersion` rpc endpoint.